content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -351,3 +458,37 @@ The label to instruct Istio to automatically include applications in the `defaul
 {{< text bash >}}
 $ kubectl label namespace default istio.io/dataplane-mode-
 {{< /text >}}
+
+{{< tabset category-name="config-api" >}}
```

Comment:
This whole tab section should not be needed. It should be done in the bookinfo sample cleanup script used in the first cleanup instruction, above.